src/components/accounts/recovery_setup/SetupRecoveryMethod.js

```diff
@@ -64,15 +67,22 @@ class SetupRecoveryMethod extends Component {
         phoneInvalid: false,
         recoverySeedPhrase: null,
         recaptchaToken: null,
-        isNewAccount: false
+        isNewAccount: false,
+        settingUpNewAccount: false
     }
 
+    emailInput = createRef()
+    phoneInput = createRef()
+
     async componentDidMount() {
         const { router, checkIsNew } = this.props;
         const { method } = router.location;
 
         if (method) {
             this.setState({ option: method });
+            if (method === 'email' || method === 'phone') {
```

Comment:
:nit:not a blocker, but at some point "soon" we should ensure that these are encapsulated in a constant somewhere we can import them from so it's immediately identifiable which files use any of these magic strings without needing to search for every possible magic string that's part of the `methods` enumeration in order to do so.